File: src/main/java/com/tannerrutgers/sechallenge/application/controller/HomeController.java
package com.tannerrutgers.sechallenge.application.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

/**
 * MVC Controller for home page
 */
@Controller
public class HomeController {

    @RequestMapping(method = RequestMethod.GET, value="/")
    public String loadPage() {
        return "index";
    }
}

File: Wave Challenge/Wave Challenge/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace Wave_Challenge.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewBag.Title = "Home Page";

            return View();
        }
    }
}
